blazor/file-manager/file-operations.md

```diff
@@ -260,7 +260,7 @@ The following table represents the response parameters of *rename* operations.
 
 |Parameter|Type|Default|Explanation|
 |----|----|----|----|
-|files|FileManagerDirectoryContent[]|-|Details of the renamed item.|
+|files|FileManagerDirectoryContent[ ]|-|Details of the renamed item.|
```

Comment:
@suba-1997 - Is this ensured in all places of UG

Reply:
will check and update in further commits.